src/odemis/driver/xt_client.py

```diff
@@ -2679,12 +2712,12 @@ def __init__(self, name, role, parent, hfw_nomag, channel="electron1", **kwargs)
     @isasync
     def applyAutoStigmator(self):
```

Comment:
Don't you wanna uncomment this line as well? https://github.com/delmic/odemis/blob/7f693ccd847c0db0d3eb2802bc44cf9867d3b776/src/odemis/gui/cont/tabs/fastem_overview_tab.py#L129

Reply:
yes, I'll do that